app/sidekiq/lighthouse/create_intent_to_file_job.rb

```diff
@@ -50,35 +50,42 @@ class InvalidITFTypeError < StandardError; end
       )
     end
 
-    ##
     # Create an Intent to File using the Lighthouse ITF endpoint for given in progress form, ICN, and PID
-    #
     # On success/failure log and increment the respective Datadog counter
+    # @see BenefitsClaims::Service#get_intent_to_file
+    # @see BenefitsClaims::Service#create_intent_to_file
     #
     # @param in_progress_form_id [Integer]
     # @param veteran_icn [String] veteran's ICN
     # @param participant_id [String] veteran's participant ID
     #
+    # @return [Hash] 'data' response from BenefitsClaims::Service
     def perform(in_progress_form_id, veteran_icn, participant_id)
       init(in_progress_form_id, veteran_icn, participant_id)
 
-      itf_log_monitor.track_create_itf_begun(itf_type, form&.created_at&.to_s, form&.user_account_id)
-
       service = BenefitsClaims::Service.new(veteran_icn)
-      service.create_intent_to_file(itf_type, '')
+      begin
+        itf_found = service.get_intent_to_file(itf_type)
+        return itf_found if itf_found.present? && itf_found['data']['attributes']['status'] == 'active'
```

Comment:
A safer version here would be `itf_found['data']&.dig('attributes', 'status') == 'active'`